lib/h_pyramid_sentry/tox.ini

```
@@ -0,0 +1,33 @@
+[tox]
+envlist = py36-tests
+skipsdist = true
+minversion = 3.8.0
+requires =
+  tox-pip-extensions
+  tox-pyenv
+tox_pip_extensions_ext_venv_update = true
+tox_pyenv_fallback = false
+
+[testenv]
+skip_install = true
+sitepackages = {env:SITE_PACKAGES:false}
+setenv = dev: PYTHONPATH = .
+passenv =
+    codecov: CI TRAVIS*
+deps =
+    {tests,lint}: .[tests]
+    lint: pylint
+    {checkformatting,format}: black
+    coverage: coverage
+    codecov: codecov
+    pip-compile: pip-tools
+whitelist_externals =
+    {lint,format,checkformatting}: sh
+commands =
+    tests: coverage run -m pytest
+    lint: sh -c "pylint {posargs:$(python setup.py  --name) tests}"
+    format: sh -c "black {posargs:$(python setup.py  --name) tests}"
+    checkformatting: sh -c "black --check {posargs:$(python setup.py  --name) tests}"
+    coverage: coverage report --show-missing
+    codecov: codecov
+    pip-compile: pip-compile
```

```suggestion
    pip-compile: {posargs:pip-compile}
```

I think we'll need to add `posargs` support to this one because we sometimes want to run `pip-upgrade` in it instead.

We should probably add the https://pypi.org/project/tox-run-command/ plugin at some point (lets you override an env's commands for one run of that env, without having to explicitly add support for that to every command in `tox.ini`). But that can wait for another time

---

I went with `pip-compile {posargs}` because I couldn't see a good reason to be doing anything else with this command than running `pip-compile`.

I still probably need to add this and other changes to the 4 other repos... :(